src/main.rs

```diff
@@ -104,14 +186,39 @@ impl MyGame {
         mb.line(&[Vec2::new(0.0, 1.0), Vec2::new(1.0, 0.0)], 0.05, Color::from_rgb(100, 100, 100));
         let cross_mesh = graphics::Mesh::from_data(ctx, mb.build());
         let game_state = grid::GameState::new(lvl_desc.into());
+
+        let x_offset = cell_num_to_coord(max_nums_in_rows);
+        let y_offset = cell_num_to_coord(max_nums_in_cols);
+
+        let game_zone = ClickableZone::new(
+            Point2::<f32>::from([x_offset, y_offset]),
+            Vector2::<f32>::from([cell_num_to_coord(game_state.width()), cell_num_to_coord(game_state.height())])
+        );
+
+
+
+        let arrow_default_color = Color::from_rgb(0, 0, 0);
+        let arrow_hover_color = Color::from_rgb(127, 127, 127);
+        let width = 0.02;
+
+        let mut undo_zone = ClickableZone::new(
+            Point2::<f32>::from([
+                0.0,
+                cell_num_to_coord(max_nums_in_cols + game_state.height())
+            ]),
+            Vector2::<f32>::from([CELL_SIZE, CELL_SIZE]),
+        );
+        undo_zone.set_mesh_for_draw(NonogramMeshBuilder::left_arrow(width, arrow_default_color, &ctx));
```

Comment:
<img src="https://github.com/alexesmet/nonogram-gamer-rs/assets/36900282/cb8404b0-336a-4c90-813e-ddcbc6b4595b" width="200">

~~Drawable mesh for clickable area can be configured in `ClickableZone::new`. There is no reason to store the mesh in the `Option`. Please remove options and setters~~

Ok ok I get it, some clickable zones should not be displayed